app/views/recipes/show.html.erb

```diff
@@ -1,10 +1,8 @@
-<p style="color: green"><%= notice %></p>
+<p class="notice-message"><%= notice %></p>
 
 <%= render @recipe %>
 
-<div>
-  <%= link_to "Edit this recipe", edit_recipe_path(@recipe) %> |
-  <%= link_to "Back to recipes", recipes_path %>
-
-  <%= button_to "Destroy this recipe", @recipe, method: :delete %>
+<div class="recipe-details">
+  <%= link_to "Edit this recipe", edit_recipe_path(@recipe), class: "edit_links" %> |
+  <%= link_to "Back to recipes", recipes_path, class: "edit_links" %>
 </div>
```

Comment:
- [ ] Great job creating the recipe show page. However, I noticed that the page didn't match the given wireframe so kindly style the page to look similar to the given wireframe :+1: 
- [ ] Kindly ensure that the `toggle button` to make a recipe public or private is implemented :+1: 
- [ ] The two other buttons also need to be implemented since a user needs to be able to `add ingredients` so that the ingredients that are missing can be `generated as the shopping list` :wink: 

| Your Design | The Given Wireframe |
|--------|--------|
| ![image](https://github.com/ridabensalem/recipe-app/assets/102699958/c277500c-12ee-49ef-80e4-be42a90a6670)|![image](https://github.com/ridabensalem/recipe-app/assets/102699958/aa8ba9ff-6f2e-441c-af7f-ab41a445a857) |

Reply:
Fixed.